Makefile

```diff
@@ -306,7 +316,7 @@ generate-release-tags:
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:crdVersions=v1"
-CONTROLLER_GEN = "$(TOOLS_BIN_DIR)/controller-gen"
+CONTROLLER_GEN = "$(TOOLS_BIN_DIR)/github.com/openshift-kni/debug-tools/cmd/knit/main.gocontroller-gen"
```

Comment:
I don't "get" this change. Can you please explain?

Reply:
editing mistake, need to be removed